docs/book/extensions/podcast-index.md

```diff
@@ -0,0 +1,22 @@
+# PodcastIndex Extension
```

Comment:
```suggestion
# PodcastIndex
```

The suffix "Extension" is not needed the documentation page contains already the section name. See the following screenshot and "Reader":

![Bildschirmfoto 2020-12-21 um 12 55 16](https://user-images.githubusercontent.com/103362/102774741-e2d42080-438b-11eb-9507-144f6dd8bdf4.png)

And the title of the page must be the same as in the navigation.

---

What is the correct spelling? "PodcastIndex" or "Podcast Index"? The following can be found on the website and GitHub:

* https://github.com/Podcastindex-org: "The Podcast Index is an open project…"
* https://podcastindex.org: "Podcast Index LLC is a software developer…"
* https://github.com/Podcastindex-org/docs-api: "Podcast Index API Documentation"

And the last paragraph of this file contains "Podcast Index".

Reply:
Yes, I was getting confused about the spelling myself. In my mind it's the `PodcastIndex` extension, just like how it's listed in the Reader docs. So when I'm talking about the extension I write `PodcastIndex` otherwise it's just Podcast Index.

Reply:
I can not found a reference for "PodcastIndex" therefore I think it must be "Podcast Index".